WebContent/js/add/job.js

```diff
@@ -94,9 +94,8 @@ function attachFormValidation() {
 	// Add regular expression capabilities to the validator
 	$.validator.addMethod(
 		"regex",
-		function(value, element, regexp) {
-			var re = new RegExp(regexp);
-			return this.optional(element) || re.test(value);
+		function(value, element, str) {
+			return element.validity.patternMismatch;

```

Comment:
I think this function is supposed to return `true` if the value is valid, and `false` otherwise.
This is doing the opposite: returning `true` if there is a `patternMismatch` and `false` if it is otherwise valid.

https://jqueryvalidation.org/jQuery.validator.addMethod/

```suggestion
			return !element.validity.patternMismatch;
```